app/src/main/kotlin/no/nav/k9/omsorgspenger/overfoerdager/OverførDagerConverter.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.k9.omsorgspenger.overfoerdager
+
+import no.nav.k9.rapid.behov.OverføreOmsorgsdagerBehov
+import java.util.Collections.singletonList
+
+internal class OverførDagerConverter {
+
+    companion object {
+        fun map(dto: OverførDagerDTO): OverføreOmsorgsdagerBehov {
+            val (journalpostId, søknad) = dto
```

Comment:
Hadde det vært enklere for "fremtiden" om det allerede nå var en liste med journalpostIder man fikk her? Ikke bare èn?

Reply:
yeah, hadde nok det. jeg endrer